packages/core/hoc/hoc.js

```diff
@@ -70,7 +70,9 @@ const hoc = (defaultConfig, hawk) => {
 		defaults = null;
 	}
 
-	return (config, maybeWrapped) => {
+	let Component;
```

Comment:
It looks better to define `Component` as `const` variable with assignment below.